library/globals.inc.php

```diff
@@ -3170,6 +3170,13 @@ function gblTimeZones()
             xl('Encryption key issued by Weno eRx service.')
         ),
 
+        'weno_md5_hash' => array(
```

Comment:
How can this be included in the install?

Reply:
hi @juggernautsei, wouldn't this be unique to each clinic and have to be hand entered by the end user?

Reply:
No, the encryption key is what is unique. The MD5 hashed password will be inside the encrypted URL that is encrypted with the encryption key. The password just can't be clear text even if it is inside the encrypted URL.

Reply:
They are discussing it now that I have raised the question. They are thinking of dropping the MD5 hash of the password. Stay tuned.